py3status/modules/window_title.py

```diff
@@ -51,7 +51,12 @@ def __init__(self):
     def window_title(self):
         window = find_focused(i3.get_tree())
 
+        if not window or window.get('name') is None or \
+           str(window.get('num')) == window.get('name'):
```

Comment:
Confirmed. This hides the workspace numbers. 

This does not address `named_workspace`. With i3's `strip_workspace_numbers yes`, I see `scratchpad` workspace along with `window_title` displaying `0:scratchpad`.

I don't know if it's a bug or just whatever bonus feature.

It does not happen in `window_title_async`. By default, it's hidden if you have visible titles. Use `always_show = True` to display them, then... with empty workspace/window, you have option to rename it to something else `♥`.